### ISSUE TITLE
Fix OneCycleLR scheduler

### ISSUE THREAD

**Issue by jedrzejwalega** [Appsilon/image_flow_cytometry_fine_tune#5]
Some bug in the code causes the scheduler's lr to constantly go up in the training regime. Need to fix it before progressing with further fine-tuning.

Exemplary [run](https://app.neptune.ai/o/appsilon/org/image-flow-cytometry-finetune/runs/details?viewId=standard-view&detailsTab=charts&shortId=IM-20&type=run) on Neptune.

**Comment by jedrzejwalega** [comment]
- OneCycleLR scheduler in lightning by default works with epochs, not steps
- If you want it to work with steps, you need to return a lr scheduler config along with the optimizer in the configure_optimizers LightningModule function
- In your Trainer, if you define a LearningRateMonitor callback and set it to logging_interval="step" the scheduler won't be smart enough on its own to adjust its updates from epoch to steps. It needs its config.

Fixed by [this](https://github.com/Appsilon/image_flow_cytometry_fine_tune/commit/972750bf62aaeda52df8aeab25c351f32e5ebd9b) commit.
